frontend/event/event_details.html

```diff
@@ -46,17 +46,29 @@ <h3 md-colors="{color:'default-teal-400'}" ng-if="!eventDetailsCtrl.endInOtherMo
                 </md-button>
               </md-menu-item>
               <md-menu-item id="hide-share">
-                <md-button ng-click="eventDetailsCtrl.share($event, eventDetailsCtrl.event)"">
+                <md-button ng-click="eventDetailsCtrl.share($event, eventDetailsCtrl.event)">
```

Comment:
there is no need to pass the event as a parameter once the controller already have its reference